spec/extractors/project_config_extractor_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe RepoAnalyzer::ProjectConfigExtractor, repo_analyzer_extractor_helpers: true do
+  describe "#extract" do
+    let(:expected) do
+      {
+        "simplecov" => false,
+        "makefile_commands" => [],
+        "rubocop_rules" => []
+      }
+    end
+
+    it { expect(extract).to eq(project_config_extractor: expected) }
+
+    context "with existent config files" do
+      let(:expected) do
+        {
+          "simplecov" => true,
+          "makefile_commands" => [
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/WordArray: Use %w or %W for an array of words.